### ISSUE TITLE
Missing "Get Computed Role" and "Get Computed Label" Commands

### ISSUE THREAD

**Issue by jginsburgn** [mozilla/geckodriver#1758]
## Description

The newest `geckodriver` (0.27.0) with the latest pre-release `Firefox` (80.0beta) implement neither the ["Get Computed Role"](https://www.w3.org/TR/webdriver2/#get-computed-role) nor the ["Get Computed Label"](https://www.w3.org/TR/webdriver2/#get-computed-label) commands from the [W3C WebDriver Level 2 Working Draft](https://www.w3.org/TR/webdriver2/).

## System

* Version: 0.27.0
* Platform: Linux Debian 9.5.0
* Firefox: 80.0beta
* Selenium: Ad-hoc clien

## Testcase

Make an HTTP request to the `/session/{session id}/element/{element id}/computedrole` endpoint, with an active session and non-stale element. The test case for the "Get Computed Label" command is analogous.

## Expected Result

Status 200 with payload:

```
{
    "value": "<computed role>"
}
```

## Actual Result

Status 405 with payload:

```
{
    "NON_JSON_CONTENT": "HTTP method not allowed"
}
```

**Comment by whimboo** [comment]
Yes, both are fairly new and haven't been implemented yet on our side. The current status can be followed via https://bugzilla.mozilla.org/show_bug.cgi?id=1585622.

**Comment by jginsburgn** [comment]
Thanks I will be on the look!

**Comment by juliandescottes** [comment]
The geckodriver bug will be https://bugzilla.mozilla.org/show_bug.cgi?id=1820782, https://bugzilla.mozilla.org/show_bug.cgi?id=1585622 only covers the marionette implementation
